content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-3.md

```diff
@@ -0,0 +1,25 @@
+---
+title: v1.8.3 Armory Operator
+toc_hide: true
+version: 01.08.03
+description: Release notes for Armory Operator v1.8.3
+date: 2023-01-23
+---
+
+## 2023-01-23 Release Notes
+
+## Security
+
+Armory scans the codebase as we develop and release software. For information about CVE scans for this release, contact your Armory account representative.
+
+## Known issues
+
+Armory Operator may fail to start in some EKS clusters. Please upgrade to Armory Operator v1.8.4.  
+
+## Highlighted updates
+
+* Support for basic installation as an alternative RBAC model
```

Comment:
Again, assuming a user would know what "Support for basic installation as an alternative RBAC model" means. I don't recall anything in the Operator docs about this, but I could be wrong

Reply:
This is enabling the basic (as opposed to cluster) level installation method that we call out in our docs *from the Helm chart* that the ATIPO team consumes. I've suggested a rewording here to make that more clear.